packages/core/src/main/cryptography/CryptographyDatabaseRepository.ts

```diff
@@ -28,6 +28,7 @@ export enum DatabaseStores {
   GROUP_IDS = 'group_ids',
   PENDING_PROPOSALS = 'pending_proposals',
   LAST_KEY_MATERIAL_UPDATE_DATES = 'last_key_material_update_dates',
+  LAST_KEY_PACKAGE_QUERY_DATE = 'last_key_package_query_date',
```

Comment:
Let's maybe avoid creating countless indexed db tables, it's going to be hard to maintain/migrate in the future. 

If we are going to create a lot of entries like this, maybe we could use `localStorage` for this. It's more light weight and it won't hold application critical informations. 
WDYT?

Reply:
Yeah, good idea!